preprocessing.py

```diff
@@ -120,6 +121,10 @@ def create_data_dir(dir_name: str) -> None:
         save_tensors(all_tensors, 'tensors')
     print('all in-sample downloads finished')
 
+    label_map = dict(zip(animes, list(range(len(animes)))))
```

Comment:
1. Do you think we might want the index as the key instead of the anime title? Since we have the index as the label in the dataset, it might be easier to call this dict if the key was also the label.
2. We might be able to shorten this line with `label_map = dict(enumerate(animes))`.
3. We might also want to add in an additional key `{11: 'noise'}` or something of the like.

Reply:
oh yeah good ideas. i'll add now